MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/model/usermodel/Plan.java

```diff
@@ -0,0 +1,207 @@
+package com.mapbox.mapboxandroiddemo.model.usermodel;
+
+/**
+ * Created by LangstonSmith on 5/9/17.
+ */
+
+public class Plan {
+  private String satellite;
+
```

Comment:
I used [http://www.jsonschema2pojo.org/](http://www.jsonschema2pojo.org/) and that's how they deliver the classes. I've removed the blank lines in all of the usermodel classes.